CHANGELOG.md

```diff
@@ -40,7 +41,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Prevent random number generation data-race for experimental rand exemplars in `go.opentelemetry.io/otel/sdk/metric`. (#5456)
 - Fix counting number of dropped attributes of `Record` in `go.opentelemetry.io/otel/sdk/log`. (#5464)
 - Fix panic in baggage creation when a member contains 0x80 char in key or value. (#5494)
-- Correct comments for the priority of the `WithEndpoint` and `WithEndpointURL` options and their coresponding environment variables in  in `go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc`. (#5508)
+- Correct comments for the priority of the `WithEndpoint` and `WithEndpointURL` options and their corresponding environment variables in `go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc`. (#5508)
```

Comment:
Please keep PRs constrainted to what they're doing. This fix should be another PR.

Reply:
This change was done while running `make codespell`, although if not required, will revert this manually.

Reply:
I'm not saying it shouldn't be fixed. Just that it should be in another PR :)

Reply:
cool! have reverted, will push alongwith the tests

Reply:
@dmathieu - I have made the changes as required. Please check and let me know if there are any suggestions..